mindsdb/integrations/handlers/palm_handler/palm_handler.py

```diff
@@ -110,28 +114,29 @@ def create_validation(target, args=None, **kwargs):
 
```

Comment:
Looking a lot better, one minor nit pick then should be good to go

One cool feature about `BaseModel` is that it supports the option to prevent extra arguments being passed. So we can remove lines 107-114.

to set up see comment above for how to implement